mico-admin/src/app/app-detail/app-detail.component.html

```diff
@@ -95,16 +95,5 @@ <h3>Application {{application?.name}}</h3>
 </mat-card>
 
 <!-- PUBLIC IPs-->
-<mat-card class="ma2">
-    <mat-card-title>
-        Public IPs:
-    </mat-card-title>
-    <mat-card-content>
-        <div *ngFor="let ip of publicIps">
-            <a target="_blank" rel="noopener" href="http://{{ip}}">
-                {{ip}}
-            </a><br />
-        </div>
-    </mat-card-content>
-
-</mat-card>
+<mico-app-detail-public-ip [application]="application">
```

Comment:
use shortname and version here instead of full application object for easier component reuse